dojo/finding/views.py

```diff
@@ -1908,11 +1907,24 @@ def finding_bulk_update_all(request, pid=None):
                             else:
                                 add_external_issue(finding, 'github')
 
+                if form.cleaned_data['notes']:
+                    logger.info('Setting bulk notes')
```

Comment:
I think this should be a `debug` log.

Reply:
Fully agree, I wanted to do the same but few lines above I've seen `info` so I have followed the pattern